.gitignore

```diff
@@ -1,4 +1,10 @@
 target
 test-ledger
-
 .idea
+examples/*
```

Comment:
Why ignoring and then negating it? Maybe it was only for your local environment to ignore some mess? :-)

Reply:
True, I initially implemented it for my mess within the examples folder. However, later I found it useful. But yes, it can be deleted if you find it unnecessary.